config/crd/bases/control.kubestellar.io_bindings.yaml

```diff
@@ -20,9 +20,9 @@ spec:
   - name: v1alpha1
     schema:
       openAPIV3Schema:
-        description: Binding is mapped 1:1 to a single BindingPolicy object. The decision
-          resource reflects the resolution of the BindingPolicy's selectors, and explicitly
-          reflects which objects should go to what destinations.
+        description: Binding is mapped 1:1 to a single BindingPolicy object. It reflects
```

Comment:
In English, "it" refers to the most recent noun phrase. In the current revision, that would be that BindingPolicy object, which is not what you are trying to say.

Reply:
I didn't find an authoritative source where pronoun references are clearly defined. However ChatGPT agrees with my feeling:
<img width="582" alt="image" src="https://github.com/kubestellar/kubestellar/assets/8633434/380ff169-0a7b-48b3-848c-ebae95312762">

But I will also respect a native speaker's suggestion and follow up in another PR.